src/cromshell/__main__.py

```diff
@@ -37,7 +39,14 @@
     flag_value=logging.NOTSET,
     help="Highest level logging for debugging",
 )
-def main_entry(verbosity):
+@click.option(
```

Comment:
I gave this a bit of more thoughts, and wonder if a user wants to specify this variable via a cmd line option, given that typically, there are many keys you want to query for, i.e. what extra value this brings in, compared to having a 

* user defined config file
* overriding through an environment var

The answer to this question applies to all these other "set up" variables (i.e. those global variables in the bash implementation).

Reply:
This was a suggestion by Louis
https://broadinstitute.slack.com/archives/G0168LSM895/p1598294907004000